ledgerReader.go

```diff
@@ -56,7 +57,16 @@ func includeFile(filename string, buf *bytes.Buffer) error {
 				return fmt.Errorf("%s:%d: invalid include directive", filename, lineNum)
 			}
 
-			err := includeFile(filepath.Join(filename, "..", pieces[1]), buf)
+			// Resolve filepaths
+			includedPath := filepath.Join(filename, "..", pieces[1])
+			includedPaths, err := filepath.Glob(includedPath)
+
+			// Include all resolved filepaths
+			for i := 0; i < len(includedPaths) && err == nil; i++ {
+				if !includedFiles[includedPaths[i]] {
```

Comment:
Is this needed (i.e. is it ever `true`), since the `includeFile` function will be removing the include from the map as part of a `defer`?  Also, it seems like having a check here would suppress the "include cycle" error?

Reply:
Hmm. This would suppress the error, but it doesn't allow it to be a problem. The use case for having this check is demonstrated in the test, where `ledgerReader_input_wildcard_root`includes "ledgerReader_input_wildcard_*". Perhaps there is another way to get around this?

Reply:
Ah yeah... I see what you mean now. I do think it would be good to preserve the error when wildcards aren't involved. I'll also give some thought on how to achieve this cleanly.